pkgs/applications/audio/parlatype/default.nix

```
@@ -0,0 +1,62 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, meson, gtk3, dbus, gst_all_1, ninja, gettext, appstream-glib, python3, desktop-file-utils, glib, gobject-introspection, gsettings-desktop-schemas, isocodes, itstool, libxml2, wrapGAppsHook, hicolor-icon-theme }:
```

One per line, almost-alphabetical - to help readability:

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, appstream-glib
, dbus
, desktop-file-utils
, gettext
, glib
, gobject-introspection
, gsettings-desktop-schemas
, gst_all_1
, gtk3
, hicolor-icon-theme 
, isocodes
, itstool
, libxml2
, meson
, ninja
, pkg-config
, python3
, wrapGAppsHook
}:
```

---

Done.